### ISSUE TITLE
Now playing from spotify

### ISSUE THREAD

**Issue by karlek** [datasektionen/Meta-TV#62]
Synchronize spotify with the telly to show the currently played song.

**Comment by danielsson** [comment]
+1

**Comment by aronstrandberg** [comment]
:+1:

**Comment by karlek** [comment]
Maybe we can convince the street team to give us a Datasektionen account?

**Comment by perhells** [comment]
Correct me if I'm wrong, but Spotify is for personal use only. Would this be a problem?

**Comment by mheine** [comment]
@perhells I don't think that would be a problem. We're not authorizing other users to play songs from an account, we're simply showing what's being played, if I understand things corrrectly :)

**Comment by perhells** [comment]
@mheine I interpreted @karlek 's entry as Datasektionen having an account to play music from. I believe the organization cannot own an account.

The idea to list which song is currently playing should be no problem. I know that Axel Samuelsson already created a page which did exactly that, along with cover arts, etc.

**Comment by aronstrandberg** [comment]
vad har du att säga till ditt försvar @limbero

**Comment by karlek** [comment]
Upstream API lacks the feature: https://github.com/spotify/web-api/issues/12

**Comment by lyxell** [comment]
One workaround could be to use last.fm scrobbling and use their api to retrieve the last scrobbled song. But this would probably require interval-polling, unless one can also get the track length via the api and set an update-event to trigger when the song has ended.
